### PR TITLE
Ft/codeforafrica posts setup

### LINKED ISSUE #510: @/codeforafrica collection Posts
### Description

Implement `Post` collection type that can be used to power `Stories` and `Opportunities`. Use @/charterafrica `News`, or `Research` as starting point but do improve it if situation necessitates.

**Note**:
1. `Post` only needs to be base/reusable in `Stories` and `Opportunities`. Users shouldn't be able to create `Post` items in the admin.
2. You'd need to create `Tag` collection as well and handle relationship between `Post` and `Tag`.
3. Would need `Author` collection as well.

Implementation should also keep in mind https://github.com/CodeForAfrica/ui/pull/443 since this approach promises to minimise changes should content editors choose to move blocks around.

Current content: https://cfa.dev.codeforafrica.org/longform/

### PR DESCRIPTION
**Pull request by kelvinkipruto**
## Description
This PR sets the building blocks required to create stories/long-form content in the CMS.

Fixes #510

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: apps/codeforafrica/src/payload/collections/Tags.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import slug from "../fields/slug";
+
+const Tags = {
+  slug: "tag",
+  admin: {
+    useAsTitle: "name",
+  },
+  access: {
+    read: () => true, // Everyone can read Pages
```
Comment (koechkevin) [inline apps/codeforafrica/src/payload/collections/Tags.js:9]:
```suggestion
    read: () => true,
```

File: apps/codeforafrica/turbo.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     },
     "build-next": {
       "outputs": [".next/**", "!.next/cache/**", "dist/**"],
-      "env": []
+      "env": ["NEXT_PUBLIC_APP_NAME", "NEXT_PUBLIC_APP_URL"]
```
Comment (kilemensi) [inline apps/codeforafrica/turbo.json:9]: Is this related to **this** PR?

**Review by kilemensi — APPROVED** [review]

**Comment by kilemensi** [comment]
@kelvinkipruto ?

File: apps/codeforafrica/turbo.json (hunk 2 above)
Comment (kelvinkipruto) [reply apps/codeforafrica/turbo.json:9]: @kilemensi No, It is caused by the merged PR #583. I did not add it there.
